### PR TITLE
Contacts API Documentation.

### PR DESCRIPTION
**Pull request by zkanda**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -246,6 +246,54 @@ We would try to find an existing contact with that phonenumber and, if none exis
...
+
+```
+
+## Updating a contact
+
+
+```
+https://test.engagespark.net/api/v1/contacts/20947107
+
+{"fullPhoneNumber":"639166752791","firstName":"Test2","organizationId":1020,"lastName":"","id":20947107,"deleted":"","carrierId":949,"phoneNumber":"9166752791","groups":[],"createdDate":"2017-10-27T03:36:30-06:00","dialingCode":"63","customFields":{"7584":"","7597":"","7754":""},"email":"","selected":0}
```
Comment (mknecht) [inline docs/index.md]: The `deleted` field cannot be part of the public API: We don't show deleted contacts. They're an internal artifact. Please discuss with @jaygel179 how to publish a public version of this API that does not include the deleted field, nor deleted contacts.

File: docs/index.md [hunk 2]
```diff
@@ -246,6 +246,54 @@ We would try to find an existing contact with that phonenumber and, if none exis
...
+Name
+
+```
+
+## Deleting a contact
+
+## Bulk-add of contacts
```
Comment (mknecht) [inline docs/index.md]: Is there something supposed to be here @zkanda ?

File: docs/index.md [hunk 3]
```diff
@@ -246,6 +246,54 @@ We would try to find an existing contact with that phonenumber and, if none exis
...
+## Custom Fields
+
+```
+https://test.engagespark.net/api/v1/customfields
+{"data": [{"organizationId": 1020, "fieldType": "text", "id": 7597, "name": "Fast"}, {"organizationId": 1020, "fieldType": "text", "id": 7584, "name": "Games"}, {"organizationId": 1020, "fieldType": "text", "id": 7754, "name": "GAMES"}]}
+```
+
+## Groups
+
+Updating of groups will be here.
```
Comment (mknecht) [inline docs/index.md]: yes?

**Comment by mknecht** [comment]
@zkanda why did you want me to review an unfinished PR?

**Comment by zkanda** [comment]
This is more of a layout on what I've been exploring in the contacts api. I just put it as a PR because you asked for it. :P

**Comment by mknecht** [comment]
OK—then please don't forget to complete it :)

**Comment by zkanda** [comment]
we don't need this for now, closing.

**Pull request closed without merging**